zone/zonedump.h

```diff
@@ -148,6 +148,7 @@ struct NPCType
 	uint16	use_model;
 	int8	flymode;
 	bool	always_aggro;
+	int     exp_mod;
```

Comment:
This should probably be uint8, because negative values have no effect, the default value is 100 and the mob already gives no EXP at value 0.